src/entries/popup/pages/settings/walletsAndKeys/walletsAndKeys.tsx

```diff
@@ -341,6 +347,33 @@ export const WalletsAndKeys = () => {
             onClick={handleCreateNewWallet}
           />
         </Menu>
+        <Menu>
+          <MenuItem
+            testId={'wipe-wallets'}
+            first
+            last
+            leftComponent={
+              <Symbol
+                size={16}
+                color="red"
+                weight="medium"
+                symbol="exclamationmark.triangle"
+              />
+            }
+            rightComponent={
+              <Symbol
+                size={16}
+                color="red"
+                weight="medium"
+                symbol="exclamationmark.triangle"
+              />
+            }
+            titleComponent={
+              <MenuItem.Title text={t('wipe_wallets.delete')} color="red" />
```

Comment:
should we hide this flow when there are no imported wallets?

Reply:
@DanielSinclair thoughts? I could see it being useful for people who have several watched wallets and want to reset.